src/components/MenuButton/index.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react"
+import Dropdown from "../Dropdown"
+
+function MenuButton (){
+  const [isMenuOpen, setIsMenuOpen] = useState(false)
+
+  function toggleMenu() {
+    setIsMenuOpen((prevIsMenuOpen) => !prevIsMenuOpen)
+  }
+
+  return (
+    <nav>
+      <button 
```

Comment:
Technically this doesn't look like the menu in the designs. However, I'll let it slide since I'd rather you spent time doing react vs more styling.